react/Pill/Pill.less

```diff
@@ -0,0 +1,87 @@
+@import (reference) '~seek-style-guide/theme';
+
+@staticTagHeight: @row-height * 5;
+@interactiveTagHeight: @row-height * 8;
+@removeCircleDiameter: @row-height * 4;
+
+.staticPill {
+  .rawText(@interaction-type-scale);
+
+  align-items: center;
+  display: inline-flex;
+  color: @sk-charcoal;
+  background: @sk-gray-light;
+  border-radius: @staticTagHeight / 2;
+  height: @staticTagHeight;
+  line-height: @staticTagHeight;
+  padding: 0 15px;
+  margin: @row-height 10px @row-height 0;
+}
+
+.interactivePill {
+  align-items: center;
+  display: inline-flex;
+  font-size: 18px;
+  background: @sk-gray-light;
+  border-radius: 24px;
+  height: @interactiveTagHeight;
+  line-height: @interactiveTagHeight;
+  padding: 0 0 0 24px;
+  margin: @row-height 10px @row-height 0;
+}
+
+.removeButton {
+  margin: 0;
+  padding: 0;
+  height: @interactiveTagHeight - 2;
+  width: @interactiveTagHeight - 2;
+  color: @sk-white;
+  border-radius: 0;
+  background: transparent;
+  appearance: none;
+  box-shadow: none;
+  border: none;
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+
+    .removeCircle {
+      .focus();
+
+      border-radius: @removeCircleDiameter / 2;
+    }
+  }
+
+  &:hover,
+  &:active {
+    .removeCircle {
+      background-color: @sk-form-accent;
+    }
+  }
+
+  &:active {
+    .removeCircle {
+      transform: scale(0.85);
+    }
+  }
+}
+
+.removeCircle {
+  background-color: @sk-mid-gray;
+  height: @removeCircleDiameter;
+  width: @removeCircleDiameter;
+  border-radius: @removeCircleDiameter / 2;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  transition: @spring-transition;
+}
+
+.removeIcon {
+  height: 12px;
```

Comment:
Curious if this needs to be exact or a percentage of the container (`.removeCircle`) as that has an enforced width

Reply:
It should be half the height of `.removeCircle` so I've changed it to `@removeCircleDiameter / 2`